Legobot/Legos/Commendations.py

```diff
@@ -29,28 +30,35 @@ def handle(self, message):
             try:
                 commendations = json.loads(json_str)
             except:
-                print('Failed to read commendations in ' + str(self) + '; initializing empty file ' + str(self.tips_file))
+                err = "Failed to read commendations in '%s'; " \
+                      "initializing empty file '%s'" % (str(self),
+                                                        str(self.tips_file))
                 commendations = {}
             try:
                 commendations[user] += commendation_amount
             except:
-                print('Could not find the user ' + str(user) + '; initializing their commendations at zero')
+                err = "Could not find the user '%s'; " \
+                      "initializing their commendations at zero" % str(user)
+                print(err)
                 commendations[user] = commendation_amount
             json_str = json.dumps(commendations)
         if json_str is not None:
             with open(self.tips_file, 'w') as f:
                 f.write(json_str)
-            metadata = Metadata(source=self, dest=message['metadata']['source']).__dict__
-            message = Message(text=(str(user) + ' given ' + str(commendation_amount) +
-                                    ' commendation points, giving them a total of ' + str(commendations[user])),
+            metadata = Metadata(source=self,
```

Comment:
Boy, even after PEP-8'ing this thing, it's still not pretty, is it? I'll have to do something about that later.